doc/examples/animation-pulsating-sphere-displacement.py

```diff
@@ -0,0 +1,74 @@
+"""Particle displacement of a pulsating sphere."""
+import sfs
+import numpy as np
+from matplotlib import pyplot as plt
+from matplotlib import animation, patches
+
+# Pulsating sphere
+center = [0, 0, 0]
+radius = 0.25
+amplitude = 0.05  # amplitude of the surface displacement
+f = 750  # frequency
+omega = 2 * np.pi * f  # angular frequency
+ka = sfs.util.wavenumber(omega) * radius
+
+# Temporal sampling for animation
+fs = f * 10  # sampling frequency
+L = int(np.round(fs / f))  # number of frames corresponding to one period
+t = np.arange(L) / fs  # time
+
+# Uniform grid
+xmin, xmax = -1, 1
+ymin, ymax = -1, 1
+grid = sfs.util.xyz_grid([xmin, xmax], [ymin, ymax], 0, spacing=0.02)
+
+# Particle displacement
+velocity = sfs.mono.source.pulsating_sphere_velocity(omega,
+                                                     center,
+                                                     radius,
+                                                     amplitude,
+                                                     grid)
+displacement = sfs.util.displacement(velocity, omega)
+
+# Animation
+fig = plt.figure(figsize=(8, 8))
+ax = plt.axes(xlim=(-xmax, xmax), ylim=(-ymax, ymax))
+patch = ax.add_patch(patches.Circle(center[:2],
+                                    radius=radius,
+                                    facecolor='RoyalBlue',
+                                    linestyle='None',
+                                    alpha=0.4))
+scat = sfs.plot.particles(grid + displacement,
+                          s=15,
+                          c='gray',
+                          marker='.')
```

Comment:
What about changing the default values of `sfs.plot.particles()` to be more sensible?

I think the only other use is in the `plot_particle_density.py` example.

Reply:
I think I don't understand what you mean by `more sensible`.
Do you mean that the particle displacement should be amplified by default, so that it is better visualized even if you use realistic values which in the order of micrometers?

Reply:
No, i don't mean the amplification. I think this should be done explicitly every single time it is used.

I meant the `marker`, `s` and `c` arguments. Because the default sizes are definitely too large for our use case, and the color black seems to be a reasonable default in our case, right?

I have never thought about using `plot()`, because in my mind this means plotting some kind of lines between connected dots, but according to the docs it does "Plot y versus x as lines and/or markers", so "lines" are not required.

I don't foresee that anyone will want to use different markers in the same plot.
But I could definitely imagine using a color gradient, e.g. for the (scalar) speed of the particles or probably for the pressure.
It might even make sense to change the size of the markers, e.g. according to sound pressure, but I didn't try this. Probably it doesn't make sense ...

I think `scatter()` is a better fit, but if you prefer `plot()`, I'm also fine with this.

Reply:
Yes, the default size is indeed too large. I would also prefer circles with black `facecolor`. I find it also nice to give some transparency (`alpha`) as this better visualizes the particle density. Should I change the default values of `sfs.plot.particles` in this PR?

I also think we can keep using `scatter`. Although the documentation says that `plot` is faster, it wouldn't be that significant in our current use cases.

By the way, visualizing the particle velocity with colors sounds nice. I'll make an example.

Reply:
> Should I change the default values of `sfs.plot.particles` in this PR?

Sure, why not!
But if you prefer, you can of course also make a new PR.